tests/test_pipeline.py

```diff
@@ -61,7 +64,7 @@ def test_predict_tails_with_novelties(self):
 
     def test_predict_heads_with_novelties(self):
         """Test scoring heads with labeling as novel w.r.t. training and testing."""
-        heads_df = self.model.predict_heads('conferences', 'brazil', testing=self.testing_mapped_triples)
+        heads_df = get_head_prediction_df(self.model, 'conferences', 'brazil', testing=self.testing_mapped_triples)
```

Comment:
I think that we also should have these tests for the `get_relation_prediction_df()` function.

Reply:
Wilco

Reply:
Done in 6f75bea0